examples/python/README.md

```diff
@@ -11,3 +14,5 @@ features:
 - support for GPU acceleration
 - batched transformation such as for videos
 - read and decode data directly as torch tensor with torchscript support (for PNG and JPEG image formats)
+
+Furhermore, previously we used to provide a very high-level API for video decoding which left little control to the user. We're now expanding that API (and replacing it in the future) with a lower-level API that allows the user a frame-based access to a video.
```

Comment:
typo: furthermore, and I would frame this differently.

This whole file will need a rewrite as it was very specific to one example
```suggestion
Furthermore, previously we used to provide a very high-level API for video decoding which left little control to the user. We're now expanding that API (and replacing it in the future) with a lower-level API that allows the user a frame-based access to a video.
```